deploy_without_migration.yml

```diff
@@ -12,4 +12,4 @@
               name: refresh_repositories
           - include_role: 
               name: deploy
-        when: deployment_allowed is succeeded
+        when: deployment_allowed.stdout.find('status') == "unlocked"
```

Comment:
doesn't it work checking the program return code or why do we have to parse stdout programatically?

Reply:
At least right now, the return code is always the same (`0`). We don't deal with potential errors: https://github.com/openSUSE/obs_github_deployments/blob/bce55230e102b6eb0f1987b27c2d3415feb9ee22/lib/obs_github_deployments/deployment.rb#L13-L20

Reply:
Okay, now I see what's the problem. You are right @dmarcoux. I'll fix the playbook and the `obs_github_deployment` to work with exit codes.

Reply:
there is nothing to change in the playbook...

Reply:
The return codes are being fixed [here](https://github.com/openSUSE/obs_github_deployments/pull/29). Once that is merged I can remove the stdout parsing.

Reply:
> there is nothing to change in the playbook...

Reverting the stdout parsing and relying on the return code alone

Reply:
> Reverting the stdout parsing and relying on the return code alone

Which is what we have today right? So closing this PR would "do the trick", no?https://github.com/openSUSE/ansible-obs/blob/master/deploy_with_migration_without_downtime.yml#L17

Reply:
https://docs.ansible.com/ansible/latest/user_guide/playbooks_error_handling.html

Reply:
> > Reverting the stdout parsing and relying on the return code alone
> 
> Which is what we have today right? So closing this PR would "do the trick", no?https://github.com/openSUSE/ansible-obs/blob/master/deploy_with_migration_without_downtime.yml#L17

So true. Closing